addon/components/bootstrap-power-select-lazy.js

```diff
@@ -13,7 +13,7 @@ export default Ember.Component.extend({
   searchByPage: undefined,
 
   /* Optional Attributes */
-  pageSize: 25,
+  debounceMS: 250,
```

Comment:
We shouldn't be overriding this... just use the branch in the `package.json`, the lib will need to be released before merging, and the `package.json` will need updated to the released version before merging.

Reply:
Crap, sorry I thought I was looking at the Denovo PR